src/Bridge/Standalone/Bootstrap.php

```diff
@@ -0,0 +1,440 @@
+use MakinaCorpus\DbToolsBundle\Database\DatabaseSessionRegistry;
+use MakinaCorpus\DbToolsBundle\Database\StandaloneDatabaseSessionRegistry;
+use MakinaCorpus\DbToolsBundle\Error\ConfigurationException;
+use MakinaCorpus\DbToolsBundle\Restorer\RestorerFactory;
+use MakinaCorpus\DbToolsBundle\Stats\StatsProviderFactory;
+use MakinaCorpus\DbToolsBundle\Storage\Storage;
+use MakinaCorpus\QueryBuilder\Error\ConfigurationError;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\LazyCommand;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Creates Symfony console application.
+ *
+ * @internal
+ * @see ../../bin/db-tools.php
+ */
+class Bootstrap
+{
+    /**
+     * Create and run Symfony console application.
+     */
+    public static function run(): void
+    {
+        static::createApplication()->run();
+    }
+
+    /**
+     * Create Symfony console application.
+     */
+    public static function createApplication(): Application
+    {
+        // @todo Test in PHAR context.
+        if (\class_exists(InstalledVersions::class)) {
+            $version = InstalledVersions::getVersion('makinacorpus/db-tools-bundle');
+        }
+        $version ??= 'cli';
+        \assert($version !== null);
+
+        $application = new Application('Db Tools', $version);
+        $application->setCatchExceptions(true);
+        $application->setDefaultCommand('list');
+
+        $definition = $application->getDefinition();
+        $definition->addOption(new InputOption('config', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Configuration files', null));
+        $definition->addOption(new InputOption('env', null, InputOption::VALUE_REQUIRED, 'Environment', 'dev'));
+
+        // Hack, we need it output to have the same configuration as the application.
+        $input = new ArgvInput();
+        $output = new ConsoleOutput();
+        (\Closure::bind(fn () => $application->configureIO($input, $output), null, Application::class))();
+
+        // We need to parse a few arguments prior running the console
+        // application in order to setup commands. This is hackish but
+        // should work.
+        $config = $configFiles = [];
+        if ($input->hasOption('config')) {
+            foreach ((array) $input->getOption('config') as $filename) {
+                $configFiles[] = $filename;
+            }
+        }
+
+        try {
+            $logger = new ConsoleLogger($output);
+            $context = self::bootstrap($config, $configFiles, $logger);
+            $defaultConnectionName = $context->databaseSessionRegistry->getDefaultConnectionName();
+        } catch (ConfigurationError $e) {
```

Comment:
This is wrong because this prevents the default `list` command of composer, and disallow options usage for choosing the config file.

Note for myself: lazy-initialize the configuration only once the commands get executed.